packages/react-aria-components/src/ToggleButton.tsx

```diff
@@ -48,7 +52,10 @@ function ToggleButton(props: ToggleButtonProps, ref: ForwardedRef<HTMLButtonElem
       slot={props.slot}
       data-pressed={isPressed || undefined}
       data-hovered={isHovered || undefined}
-      data-focus-visible={isFocusVisible || undefined} />
+      data-focused={isFocused || undefined}
+      data-focus-visible={isFocusVisible || undefined}
+      data-selected={state.isSelected || undefined}
+      data-disabled={props.isDisabled || undefined} />
```

Comment:
I think button documents these as `:focus` and `:disabled` rather than data attrs, and `isSelected` is documented using `aria-pressed` above.